explorer/client/README.md

```diff
@@ -10,6 +10,26 @@ In the project directory, run:
 
 Before running any of the following scripts `npm i` must run in order to install the necessary dependencies.
 
+# How to Switch Environment
+
+The purpose of the SuiExplorer Client is to present data extracted from a real or theoretical Sui Network.
+
+What the 'Sui Network' is varies according to the environment variable `REACT_APP_DATA`.
+
+When running most of the below npm commands, the SuiExplorer Client will extract and present data from the Sui Network connected to the URL https://demo-rpc.sui.io.
+
+If the environment variable `REACT_APP_DATA` is set to `static`, then the SuiExplorer will instead pull data from a local, static JSON dataset that can be found at `./src/utils/static/mock_data.json`.
+
+For example, suppose we wish to locally run the website using the static JSON dataset and not the API, then we would run the following:
+
+```bash
+$ REACT_APP_DATA=static npm start
```

Comment:
I would remove the dollar sign as it gets picked up by the auto-copy functions of both GitHub and our Dev Portal. We include it only when output is also included in the same code block. (I recommend splitting those out anyway.)

Reply:
Thanks Clay. I've removed the dollar sign.